source/common/protobuf/type_util.cc

```diff
@@ -0,0 +1,19 @@
+#include "common/protobuf/type_util.h"
+
+namespace Envoy {
+namespace Config {
```

Comment:
This shouldn't be in the config namespace, but instead in some protobuf prefixed one., e.g. `ProtobufTypeUril`

Reply:
Other helper function and class in protobuf repo don't have a nested namespace because Protobuf namespace is reserved for standard Protobuf library. I will just remove the Config namespace.